libraries/windows_privileged.rb

```diff
@@ -20,24 +20,13 @@
 # limitations under the License.
 #
 
-if RUBY_PLATFORM =~ /mswin|mingw32|windows/
-  require 'windows/error'
-  require 'windows/registry'
-  require 'windows/process'
-  require 'windows/security'
-end
-
 #helpers for Windows API calls that require privilege adjustments
 class Chef
   class WindowsPrivileged
-    if RUBY_PLATFORM =~ /mswin|mingw32|windows/
-      include Windows::Error
-      include Windows::Registry
-      include Windows::Process
-      include Windows::Security
-    end
     #File -> Load Hive... in regedit.exe
     def reg_load_key(name, file)
+      load_deps
```

Comment:
Is there an additional cost to re-require and include on every call?
If so, do something along the lines of @loaded ||= load_deps ?